src/Screens/index.tsx

```diff
@@ -0,0 +1,10 @@
+export type RootStackParamList = {
+  PokemonIndex: undefined;
+  Pokemon: { name: string };
+};
+
+declare global {
```

Comment:
I would create file called sth like navigationTypes.ts (instead of `index`). And avoid declaring global type